src/components/BooksPage.js

```diff
@@ -1,34 +1,34 @@
 import { useSelector, useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
 import Book from './Book';
-import { addBook } from '../redux/books/book';
+import { addBooks } from '../redux/books/book';
 
 const BooksPage = () => {
   const dispatch = useDispatch();
   const submitForm = (e) => {
     e.preventDefault();
     const title = e.target.querySelector('#title');
-    const author = e.target.querySelector('#author');
+    const category = e.target.querySelector('#category');
     const newBook = {
-      id: uuidv4(),
+      item_id: uuidv4(),
       title: title.value,
-      author: author.value,
+      category: category.value,
     };
     title.value = '';
-    author.value = '';
-    dispatch(addBook(newBook));
+    category.value = '';
+    dispatch(addBooks(newBook));
   };
   const bookList = useSelector((state) => state.books);
 
   return (
     <div>
       <h1> Awesome Books </h1>
       <ul>
-        {bookList.map((book) => <Book key={book.id} book={book} />)}
+        {bookList.map((book) => <Book key={book.item_id} book={book} />)}
       </ul>
       <form id="add-book-form" onSubmit={(e) => submitForm(e)}>
-        <input id="title" placeholder="Book Title" />
-        <input id="author" placeholder="Book Author" />
+        <input id="title" placeholder="Name" />
+        <input id="category" placeholder="Category" />
         <button type="submit"> Add Book </button>
       </form>
     </div>
```

Comment:
Please take a look at the image below
- [ ] I have deleted one book but it is still on the redux store and on the page. As specified in the project guidelines, you should make async requests that persist on the server. You seem to have done this correctly. However, your changes are not stored correctly on the redux store when I add or delete a book. I need to reload the browser for the store to be updated from the server.
  
  Remember, this project is merely a refactor of the previous one and so just like your previous `create-remove` project, the books list should be updated without page reload. Kindly fix this behavior.

  ![somdo-books](https://user-images.githubusercontent.com/36542986/145052653-45940f57-6de6-4581-b662-79ff745dc626.jpg)